python/tvm/relay/analysis/sparse_conv2d.py

```diff
@@ -108,19 +112,28 @@ def process_params(expr, params, block_size, sparsity_threshold, layout):
                 )
             else:
                 sparse_weight_data = sparse_weight.data
+        elif weight_kernel[0] == kernel_size == 3 and layout == "NHWC":
```

Comment:
Only support NHWC here?

Reply:
Sorry to miss NCHW support, will be added in a later commit. In our current testing pipeline we don't use this to convert our models, though.

Reply:
Oh, that's fine, maybe add a TODO here or throw a warning on NCHW in case others have any confusion.